cli/main.go

```diff
@@ -421,6 +430,7 @@ func main() {
 					&cli.StringFlag{Name: "host", Usage: "use customized hostname of srun4000"},
 					&cli.BoolFlag{Name: "insecure", Usage: "use http instead of https"},
 					&cli.BoolFlag{Name: "keep-online, k", Usage: "keep online after login"},
+					&cli.IntFlag{Name: "ac-id", Usage: "use specified ac_id"},
```

Comment:
May also add such a flag for `deauth` since `buildLoginParams` needs acid when logout.
Do not know the behavior when logout without correct ac_id.
According to the previous behaviour, incorrect acid does not harm.